TaskWithMaxMinSortForArray/TaskWithMaxMinSortForArray.cpp

```diff
@@ -0,0 +1,59 @@
+﻿#include <iostream>
+
+int searchMax(int array[], size_t len) {
+	int max = array[0];
+	for (int i = 0; i < len; i++)
+	{
+		if (array[i] > max)
+		{
+			max = array[i];
+		}
+	}
+	return max;
+}
+
+int searchMin(int array[], size_t len) {
+	int min = array[0];
+	for (int i = 0; i < len; i++)
+	{
+		if (array[i] < min)
+		{
+			min = array[i];
+		}
+	}
+	return min;
+}
+
+void sortArray(int array[], size_t len) {
+	for (int i = 0; i < len - 1; i++) {
+		for (int j = 0; j < len - i - 1; j++) {
+			if (array[j] > array[j + 1]) {
+				int temp = array[j];
+				array[j] = array[j + 1];
+				array[j + 1] = temp;
+			}
+		}
+	}
+}
+
+int main(int argc, char* argv[])
+{
+	if (argc == 1)
+	{
+		std::cout << "No command line arguments " << std::endl;
+		return 1;
+	}
+	int* array = new int[argc - 1];
```

Comment:
Please declare a constant to store an array size and use this down the code. There are redundant calculations of argc - 1. That's good when the compiler can optimize it out. If it couldn't such a calculation can be critical to performance in some areas.

Reply:
I did this, I left the previous working version in the comments